storage/src/cache/mod.rs

```diff
@@ -210,13 +209,13 @@ pub trait BlobCache: Send + Sync {
     /// entry in the `chunks` array in corresponding order.
     ///
     /// This method returns success only if all requested data are successfully fetched.
-    fn read_chunks(
+    fn read_chunks_from_backend(
```

Comment:
Nit: the above comment needs to be updated as well.

`Function read_chunks() returns one buffer `
to
`Function read_chunks_from_backend() returns one buffer `

Reply:
fixed